### ISSUE TITLE
How to give a single input to the trained ANN model? I am getting an error of "Matrix size-incompatible: In[0]: [4,1], In[1]: [4,3000] 	 [[{{node sequential/dense/BiasAdd}}]]" for last 5 lines of this code

### ISSUE THREAD

**Issue by sanket10101** [tensorlayer/srgan#215]
# Part 1 - Data Preprocessing
# Importing the libraries
import numpy as np
import pandas as pd
import tensorflow as tf
tf.__version__

# Importing the dataset
dataset = pd.read_excel('ANN dissertation 2.xlsx')

# Assigning Variables 
X = dataset.iloc[:, :-1].values
y = dataset.iloc[:, -1].values

# Splitting the dataset into the Training set and Test set
from sklearn.model_selection import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.2, 
                                                    random_state = 0)

# Part 2 - regression ANN creation

# Initialising the ANN
ann = tf.keras.models.Sequential()

# Adding the input layer and the first hidden layer
ann.add(tf.keras.layers.Dense(units=3000, activation='selu'))

# Adding the hidden layers
ann.add(tf.keras.layers.Dense(units=3000, activation='selu'))

ann.add(tf.keras.layers.Dense(units=3000, activation='selu'))

ann.add(tf.keras.layers.Dense(units=3000, activation='selu'))

ann.add(tf.keras.layers.Dense(units=3000, activation='selu'))

ann.add(tf.keras.layers.Dense(units=3000, activation='selu'))

# Adding the output layer
ann.add(tf.keras.layers.Dense(units=1))

# Compiling the ANN
ann.compile(optimizer = 'AdaDelta', loss = 'mean_squared_error')

# Fitting the ANN to the Training set
ann.fit(X_train, y_train, batch_size =9, epochs = 50)

# Part 3 - Making the predictions and evaluating the model

# Predicting the Test set results
y_pred = ann.predict(X_test)
np.set_printoptions(precision=2)
print(np.concatenate((y_pred.reshape(len(y_pred),1), 
                      y_test.reshape(len(y_test),1)),1))

# Single prediction:
x_new_pred = [[0.85, 34.8, 1.1, 190]]
y_new_pred = ann.predict(x_new_pred)
print(y_new_pred)
